cranelift-entity/src/set.rs

```diff
@@ -33,6 +33,14 @@ where
         }
     }
 
+    /// Creates a new empty set with the specified capacity.
+    pub fn with_capacity(capacity: usize) -> Self {
+        Self {
+            elems: Vec::with_capacity(capacity),
```

Comment:
`EntitySet` stores bits, thus the capacity should be `(capacity + 7) / 8`, to match the number of elements. (see `EntitySet::resize`)